proxy-node-gateway/src/test/java/uk/gov/ida/notification/proxy/EidasSamlParserProxyTest.java

```diff
@@ -47,14 +53,17 @@ public void shouldReturnEidasSamlParserResponse() throws Exception {
 
         assertEquals("request_id", response.getRequestId());
         assertEquals("issuer", response.getIssuer());
-        assertEquals("pub_enc_key", response.getConnectorPublicEncryptionKey());
+        assertEquals(new X509CertificateFactory().createCertificate(UNCHAINED_PUBLIC_CERT), response.getConnectorEncryptionPublicCertificate());
         assertEquals("destination", response.getDestination());
     }
-
     private EidasSamlParserProxy setUpEidasSamlParserService(String url) throws Exception {
+        ObjectMapper objectMapper = new ObjectMapper();
+        SimpleModule testModule = new SimpleModule("TestModule", Version.unknownVersion());
+        testModule.addSerializer(new X509CertificateSerializer());
+        testModule.addDeserializer(X509Certificate.class, new X509CertificateDeserializer());
```

Comment:
What's this `testModule` doing? It seems to get instantiated but not used anywhere. In the tests for the serialiser itself it looks like it gets registered on the ObjectMapper: `objectMapper.registerModule(testModule)`

Reply:
It works without the module. Go figure 🤷‍♂️